src/test/resources/com/jpinpoint/perf/lang/java/ruleset/spring/xml/AvoidModelMapAsRenderParameter.xml

```diff
@@ -25,8 +26,7 @@ public class AvoidModelMapAsRenderParameterTest {
 
 	// ModelMap in session between action and render, and after that
 	// render in method name, no RenderRequest param
-	// must flag [--> NOT, see  JPCC-88?]
-	@ResourceMapping("subview-init")
+	@ResourceMapping("subview-init") // bad [--> NOT, see JPCC-88?]
```

Comment:
Seems to me that this is found out not to be bad, see Jira issue JPCC-88. I think jira is shut down, I didn't find it.

Reply:
and I cannot remember